lib/zendesk_apps_tools/common.rb

```diff
@@ -10,6 +41,10 @@ def say_error(msg)
     end
 
     def get_value_from_stdin(prompt, opts = {})
+      if respond_to?(:options) && options[:unattended]
+        say_error 'Would have prompted for a value interactively, but we are running unattended.'
+        say_error_and_exit prompt
```

Comment:
I disgaree. We're not ignoring, we're crashing. Also, that's going to output some really long lines.